lib/enum/value.rb

```diff
@@ -0,0 +1,144 @@
+module Enum
+  class Value
+    
+    # TODO: Perhaps stored_value should be renamed unsafe_value?
```

Comment:
Let's call it `atom`, wdyt? The idea borrow from [this](http://erlang.org/doc/reference_manual/data_types.html#id62168).

Reply:
Ok, I like atom.